.github/workflows/test-python.yml

```diff
@@ -0,0 +1,28 @@
+name: pyTest for Python code
+'on':
+  pull_request: {}
+  workflow_dispatch: {}
+jobs:
+  test:
```

Comment:
change to `py-test`, we need distinct name when defining checks in branch

Reply:
changed